app/models/reminder.js

```diff
@@ -3,5 +3,5 @@ import DS from 'ember-data';
 export default DS.Model.extend({
   title: DS.attr('string'),
   body: DS.attr('string'),
-  date: DS.attr('date'),
+  date: DS.attr('string'),
```

Comment:
True story - my fault here, its still ok to use a string to get it working here.